### PR TITLE
Improve Stream Updating & Fix Memory Leak

### PR DESCRIPTION
**Pull request by tomstitt**
Use new `emglvis.updateVisualization` from https://github.com/GLVis/glvis/pull/143 to handle socket updates so that keys are preserved.
Adds "Pause/Resume" button to the "Setup" menu.
Rename `m2w.py` to `glvis-browser-server`
Fix memory leak in `startVisualization`

This PR address the two "Major" issues in https://github.com/GLVis/glvis-js/pull/1

- [x] Update *glvis.js* once https://github.com/GLVis/glvis/pull/143 is merged

### CODE REVIEW

File: glvis-browser-server [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# This file is part of the GLVis visualization tool and library. For more
+# information and source code availability see https://glvis.org.
+#
+# GLVis is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+# The glvis-browser-server script is an MFEM socket-to-websocket forwarder to
+# make it easier to use glvis-js as a viewer for inline MFEM visualization.
+#
```
Comment (tzanio) [inline glvis-browser-server:16]: I like the new name!

File: examples/websockets.html [hunk 2]
```diff
@@ -451,10 +455,30 @@ <h4 align="center">Stream update delay</h4>
         document.getElementById(id).style.width = "0";
       }
 
+      // pause bottom for streams
```
Comment (tzanio) [inline examples/websockets.html:458]: "pause button"?

File: examples/websockets.html [hunk 3]
```diff
@@ -451,10 +455,30 @@ <h4 align="center">Stream update delay</h4>
...
+      var stream_paused = false;
+      var pause_button = document.getElementById("pause-btn");
+      function setToggleStreamPauseText() {
+        if (stream_paused) {
+          pause_button.innerHTML = "Resume";
+        } else {
```
Comment (tzanio) [inline examples/websockets.html:464]: Can we use this to display a "loading" spinner? Some of the data files are big/interesting and starting to take a few seconds

File: glvis-browser-server (hunk 1 above)
Comment (tomstitt) [reply glvis-browser-server:16]: credit to @v-dobrev !

File: examples/websockets.html (hunk 3 above)
Comment (tomstitt) [reply examples/websockets.html:464]: I like that idea, I'll improve in the target branch

**Pull request merged**